src/components/Vuetify/index.ts

```diff
@@ -41,6 +41,16 @@ const Vuetify: VuetifyPlugin = {
         rtl: opts.rtl,
         theme: theme(opts.theme)
       },
+      watch: {
+        rtl (value: boolean) {
+          document.documentElement.dir = value ? 'rtl' : 'ltr'
+        }
+      },
+      created () {
+        // To set the html elements 'dir' attribute initially in client.
+        // TODO: also set initial 'dir' when using SSR.
+        if (opts.rtl !== undefined && document) this.rtl = this.rtl
```

Comment:
This will break SSR, and `immediate: true` would do the same thing.

Reply:
I change this in new commit, also test it with vuetify.com and worked. is there any problem in new approach?